### ISSUE TITLE
Toggling Plex Oauth breaks API.

### ISSUE THREAD

**Issue by slimshizn** [tidusjar/Ombi#2232]
#### Ombi build Version:

V 3.0.3239 (Plex Oauth added) - 3.0.3293 Current

#### Update Branch:

Open Beta

#### Media Sever:

Plex

#### Operating System:

Unraid 6.5.1

#### Ombi Applicable Logs (from `/logs/` directory or the Admin page):

2018-05-06 23:05:22.147 -04:00 [Error] StatusCode: Unauthorized, Reason: Unauthorized, RequestUri: https://plex.tv/users/sign_in.json
2018-05-06 23:05:53.624 -04:00 [Error] StatusCode: Unauthorized, Reason: Unauthorized, RequestUri: https://plex.tv/users/sign_in.json
2018-05-06 23:07:00.799 -04:00 [Error] StatusCode: Unauthorized, Reason: Unauthorized, RequestUri: https://plex.tv/users/sign_in.json
2018-05-06 23:07:38.973 -04:00 [Error] StatusCode: Unauthorized, Reason: Unauthorized, RequestUri: https://plex.tv/users/sign_in.json
2018-05-06 23:08:53.498 -04:00 [Error] StatusCode: Unauthorized, Reason: Unauthorized, RequestUri: https://plex.tv/users/sign_in.json

#### Problem Description:

Enabled Plex Oauth to check out the new feature, disabled after not being needed. API worked prior to doing so. API continues to be broken as shown in Postman. 

#### Reproduction Steps:

Shown above.

**Comment by slimshizn** [comment]
Not sure if those logs are part of the api issue, but they are the only logs I could find that had any errors other than not finding recorded tv shows errors. 
![2018-05-07_07-55-22](https://user-images.githubusercontent.com/23506499/39701242-6dcbe0ea-51ce-11e8-87d3-46ac660d191e.png)
![2018-05-05_10-59-53](https://user-images.githubusercontent.com/23506499/39701283-94a8d3ee-51ce-11e8-9183-06709e1ed664.png)
DB API is also different than UI API.

**Comment by slimshizn** [comment]
![2018-05-05_11-03-32](https://user-images.githubusercontent.com/23506499/39701318-b908f75a-51ce-11e8-8826-085d720877a2.png)
Entering DB API results in invalid.

**Comment by tidusjar** [comment]
The first part is that you are not authorized via plex. I have no control over that. 

I also cannot reproduce the other issues. 

Now you see the ApiKey in the db, that's what the ui uses, so if you are seeing different in the UI then you might be looking at the wrong database

**Comment by slimshizn** [comment]
I'm deleting everything and starting over fresh

**Comment by tidusjar** [comment]
Cannot reproduce and then realised that you have `Hide Requests From Other Users` enabled. That's the issue here.

**Comment by slimshizn** [comment]
Yes that was the issue, can confirm. Why is that shutting off API?

**Comment by tidusjar** [comment]
Because when you used the API key it's technically a user behind the scenes. It just the logic was not excluding "System Users". 

Reason why it's a user is for dB integrity, each request has a foreign key to a user.
